content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -100,8 +100,8 @@ afterwards to tell Kubernetes that it can resume scheduling new pods onto the no
 
 ## Draining multiple nodes in parallel
 
-The `kubectl drain` command should only be issued to a single node at a
-time. However, you can run multiple `kubectl drain` commands for
+The `kubectl drain` command should only be issued to a single node at a time. This is because as maintenance is being performed on the node, the pods are safely evicted until the node is back in service. If the command is used for more than a single node, the pods that are part of the service may also get terminated due to eviction. This can cause disruption and potential data loss in the running application.
```

Comment:
> This is because as maintenance is being performed on the node, the pods are safely evicted until the node is back in service.

There is no indication of the node being out of service, right?
What if I'm draining a node for other reasons?

> If the command is used for more than a single node, the pods that are part of the service may also get terminated due to eviction.

The command can be triggered on different nodes, as explained later in this paragraph.
What are you trying to clarify? Which service are you referring to?

> This can cause disruption and potential data loss in the running application.

Is this something you want to clarify?

Overall, I'm not quite clear what value this PR is adding.

Reply:
> Is this something you want to clarify?

Yess !! Here I provided a brief idea about how draining multiple nodes in parallel can affect the application. Explaining each and every scenario can make it little too long? Is there any better way you can guide me through this....if i am not on the right path

Reply:
My suggestions would be: clarify whatever you want to clarify while skip things you are not 100% sure.

Reply:
I agree with @tengqm. If you are trying to clarify the issue of a potential loss when draining nodes in parallel, you should focus on that alone.

@Shubhdeep02, you need to realize that this section focuses on draining multiple nodes in parallel while also considering that a PodDisruptionBudget policy is applied. This policy ensures that a minimum number of pods in a deployment or replica set remain available during voluntary disruptions like executing a `kubectl drain` command.

Since not applying a PodDisruptionBudget policy in this scenario isn't a best practice, you could maybe outline at the end of the section that applying this policy is a best practice, and not doing so can cause disruption and potential data loss in the running application.